MainApp/.gitignore

```diff
@@ -8,4 +8,4 @@ tmp/
 !.gitkeep
 dist
 .idea/
-public/downloads/*
```

Comment:
I'm not sure why gamedev commits are showing here. But I'm removing the csvs and adding public/downloads/* to gitignore and committing.